### ISSUE TITLE
KSQL in headless mode doesn't do the same validation on input queries as the interactive mode

### ISSUE THREAD

**Issue by apurvam** [confluentinc/ksql#1320]
In non-interactive mode, we accept statements like the following where we create a stream for an aggregation. 

Say we have a `queries.sql` file with the following contents:

```sql
CREATE STREAM pageviews_original_2 (viewtime bigint, userid varchar, pageid varchar) WITH (kafka_topic='pageviews_kafka_topic_json', value_format='json');
CREATE STREAM pageview_counts as select pageid, count(*) FROM pageviews_original_2 GROUP BY pageid;
```

We can then run this script just fine. It _should_ fail because we are trying to create a stream where instead we should be create a table.

```
amehta-macbook-pro-2:confluent-5.0.0-beta1 apurva$ ./bin/ksql-server-start  -daemon ./etc/ksql/ksql-server.properties --queries-file queries.sql
amehta-macbook-pro-2:confluent-5.0.0-beta1 apurva$ ./bin/kafka-topics --zookeeper localhost:2181 --list
PAGEVIEW_COUNTS
__confluent.support.metrics
__consumer_offsets
_confluent-ksql-default__command_topic
_confluent-ksql-default_query_CTAS_PAGEVIEW_COUNTS-KSTREAM-AGGREGATE-STATE-STORE-0000000006-changelog
_confluent-ksql-default_query_CTAS_PAGEVIEW_COUNTS-KSTREAM-AGGREGATE-STATE-STORE-0000000006-repartition
_schemas
connect-configs
connect-offsets
connect-statuses
pageviews_kafka_topic_json
amehta-macbook-pro-2:confluent-5.0.0-beta1 apurva$ ./bin/kafka-console-consumer --bootstrap-server localhost:9092 --topic PAGEVIEW_COUNTS
{"PAGEID":"Page_51","KSQL_COL_1":3}
{"PAGEID":"Page_90","KSQL_COL_1":1}
^CProcessed a total of 2 messages
```

In interactive mode, this same pair of statements does not execute.

```
ksql> create stream pageviews_original (viewtime bigint, userid varchar, pageid varchar) WITH (kafka_topic='pageviews_kafka_topic_json', value_format='json');

 Message
----------------
 Stream created
----------------
ksql> select * from pageviews_original LIMIT 1;
1526596419700 | 3361 | 1526596419700 | User_2 | Page_13
LIMIT reached for the partition.
Query terminated
ksql> create stream pageview_counts as select pageid, count(*) from pageviews_original group by pageid;
Invalid result type. Your SELECT query produces a TABLE. Please use CREATE TABLE AS SELECT statement instead.
```

**Comment by big-andy-coates** [comment]
This PR is looking to start to consolidate the rest app with the standalone executor: https://github.com/confluentinc/ksql/pull/1897

**Comment by apurvam** [comment]
This should be fixed by https://github.com/confluentinc/ksql/pull/1599

**Comment by big-andy-coates** [comment]
Not fixed by #1599

**Comment by big-andy-coates** [comment]
This is now fixed in master
